### ISSUE TITLE
Miskelly - Delivery Discount rule

### ISSUE THREAD

**Issue by i-rabot** [kbower/tickettest1#3416]
_Imported from trac ticket 3416.
Created by **aadams**
Opened in trac: Nov 5, 2015, 10:46:37 AM
Last modified in trac: Dec 8, 2015, 2:57:27 PM_
***
_by aadams:_
We are running a promotion that starts Monday,  we are offering free delivery for 6 specific zip codes and only if they bring in the mailer, this is for a defined period of time and Same Day Delivery does not qualify.  How can we set this up so the system will not charge a delivery fee?  Possibly with a rule?   
___
This is all spelled out in #2366.  Notes are there for how to complete this request.

**Comment by i-rabot** [comment]
_**sjohnson** trac comment on Nov 5, 2015, 10:54:15 AM:_

Same day delivery = zone SDD

**Comment by i-rabot** [comment]
_**kbower** trac comment on Nov 10, 2015, 9:20:37 AM:_

Replying to [sjohnson](comment:1):
> Same day delivery = zone SDD 


There is a pretty major difference between this ticket and #2366.  For this request, the discount on delivery is "only if they bring in the mailer."  That makes this not automatic and only barely related to #2366.

Further, since manual intervention is required for telling Loft "this customer has the mailer," that manual intervention might as well be overriding the delivery price to $0.  (The intent is probably for a non-manager to be able to do it, but that's asking for a pretty major rule implementation for this one-off discount.)

I'm close to closing this ticket with "wontfix" if they don't reply pretty soon.

**Comment by i-rabot** [comment]
_**kbower** trac status change on Nov 10, 2015, 9:20:37 AM:_ **waiting**

**Comment by i-rabot** [comment]
_**sjohnson** trac comment on Nov 10, 2015, 9:44:37 AM:_

What are we waiting on?  I don't know of any outstanding questions.

**Comment by i-rabot** [comment]
_**kbower** trac comment on Nov 10, 2015, 10:07:57 AM:_

https://retailarchitects.freshdesk.com/helpdesk/tickets/1327

**Comment by i-rabot** [comment]
_**kbower** trac status change on Dec 8, 2015, 2:57:27 PM:_ **closed**
